### DISCUSSION TITLE
ImageMagick Installation Issue

### DISCUSSION THREAD

**Discussion by programmerarunkumar** [ImageMagick/ImageMagick#7248] [Help]
### ImageMagick version

7.1.1-31

### Operating system, version and so on

OS : macOS Monterey, version : 12.5,  chip : Apple MI Pro

### Description

Im trying to install the ImageMagick from source as given in the https://imagemagick.org/script/install-source.php

Im getting the below issue while using the make command
![image](https://github.com/ImageMagick/ImageMagick/assets/58946415/c5fd31b7-78e2-4ddd-a22d-02e14876a17b)

**Comment by dlemstra** [comment]
We have made some changes in our `Makefile.am` files and this should be resolved in a couple days.

**Reply by programmerarunkumar** [reply]
Thank you
